joern-cli/frontends/rubysrc2cpg/src/main/antlr4/io/joern/rubysrc2cpg/parser/RubyParser.g4

```diff
@@ -223,8 +223,8 @@ expressions
 // --------------------------------------------------------
 
 block
-    :   braceBlock
-    |   doBlock
+    :   braceBlock                                                                                                              # braceBlockBlock
```

Comment:
u really want all this whitespace?

Reply:
Not really, but this way they are aligned throughout the entire file and somehow that feels comfy...?

Reply:
lol, ok, as long as it feels comfy... thumbs up